source/documentation/github/set-up-github.md

```diff
@@ -44,17 +44,39 @@ To create an SSH key in JupyterLab, follow the steps below:
 
 8.  Select the SSH key and copy it to the clipboard by pressing Ctrl+C on windows or ⌘C on Mac.
 
+#### Visual Studio Code
+
+To create an SSH key in Visual Studio Code, follow the steps below:
```

Comment:
This conflicts with https://user-guidance.analytical-platform.service.justice.gov.uk/get-started.html#create-and-add-visual-studio-code-ssh-key-to-github

Reply:
Great spot. I didn't know the instructions were there too. Is it worth trimming this page then that has SSH information on: https://user-guidance.analytical-platform.service.justice.gov.uk/github/set-up-github.html#set-up-github

Reply:
Actually, no harm in having both: a quick start and something under a more structured folder; I'll update this PR tomorrow with the information from the quick start so they conform.

Reply:
@jacobwoffenden I've included changes in [8b1ff69](https://github.com/moj-analytical-services/user-guidance/pull/420/commits/8b1ff6943cab789c95aa09ac2667c70c4aa679c9) which matches the SSH guidance across both pages now for all (RStudio, JupyterLab and VS Code)